src/app/chat/views/message_list.nim

```diff
@@ -47,6 +47,7 @@ QtObject:
       of ChatMessageRoles.UserName: result = newQVariant(message.userName)
       of ChatMessageRoles.Message: result = newQVariant(message.message)
       of ChatMessageRoles.Timestamp: result = newQVariant(message.timestamp)
+      of ChatMessageRoles.Clock: result = newQVariant($message.clock)
```

Comment:
What's the difference between Clock and Timestamp?

Reply:
If i understood correctly the status-go code, Clock refers to a Lamport timestamp: 
https://en.wikipedia.org/wiki/Lamport_timestamp which should be used to order the msgs